### PR TITLE
[#17] Add GitHub templates

### PR DESCRIPTION
**Pull request by ccjernigan**
These templates are more or less copied from the Zcash Android SDK with minor modifications (removing reference to screenshots, Android, or UI manual tests)

### CODE REVIEW

**Review by HonzaR — COMMENTED** [review]
Seems good to me. Just a three little things on that:

- found there is a link to `Architecture.md` file from `CONTRIBUTING.md` file, but `Architecture.md` is missing.

- I was on adding this empty `Architecture.md` file to the repository by commit, but I miss WRITE access to this repo. Please, can I get it?

- Links to the documentation files under the `.github` folder seem to be broken, but the same situation appeared on our other repositories too, and we said if they work while filing issues, then it is alright, so the same applies here imho.

**Comment by ccjernigan** [comment]
> 

I'll delete the reference to Architecture.md for now.  That'll require a bit more work to refactor the existing docs, which I'll plan to do in a followup PR.

Yes, agree that we mostly care about the links working when filing issues or opening PRs.  I think the links will be broken until we have the first version of this merged to the master branch.

**Review by HonzaR — APPROVED** [review]

**Pull request merged**
